cli/actions/setup.js

```diff
@@ -86,6 +86,9 @@ module.exports = (config, flags, args) => {
       commands = uniq(commands.concat(tempScripts));
     }
 
+    // This is the default test script added by 'npm init'.
+    const npmInitDefaultTestScript = 'echo \"Error: no test specified\" && exit 1'; // eslint-disable-line no-useless-escape
```

Comment:
Hi @jaredmcdonald, the `npm init` default test script has the escape characters in it, and I wanted to check for that string exactly.  I think this may be a valid use case for ignoring the lint rule on this line.